examples/client/dumpStop.ts

```diff
@@ -0,0 +1,44 @@
+import { createConcordiumClient } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn ts-node <path-to-this-file> [options]
+
+  Options
+    --help,     -h  Displays this message
+    --endpoint, -e  Specify endpoint of the form "address:port", defaults to localhost
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            endpoint: {
+                type: 'string',
+                alias: 'e',
+                default: 'localhost:20000',
+            },
+        },
+    }
+);
+
+const [address, port] = cli.flags.endpoint.split(':');
+const client = createConcordiumClient(
+    address,
+    Number(port),
+    credentials.createInsecure(),
+    { timeout: 15000 }
+);
+
+if (cli.flags.h) {
+    cli.showHelp();
+}
+
+/// Stop dumping packages. Only enabled if the node was built with the
+/// `network_dump` feature. Rejects if the network dump failed to be stopped.
+
+(async () => {
+    await client.dumpStop();
```

Comment:
Why is nothing printed here?

Reply:
Because the call doesn't return anything to print, but I'll make it print "Dump successful" for clarity.